lib/handlers/quote/quote.ts

```diff
@@ -651,14 +651,9 @@ export class QuoteHandler extends APIGLambdaHandler<
   static protocolsFromRequest(
     chainId: ChainId,
     requestedProtocols: string[] | string | undefined,
-    isMobileRequest: boolean,
-    appVersion: string | undefined,
     forceCrossProtocol: boolean | undefined
   ): Protocol[] | undefined {
-    // We will exclude V2 if isMobile and the appVersion is not present or is lower than 1.24
-    const semverAppVersion = semver.coerce(appVersion)
-    const fixVersion = semver.coerce('1.24')!
-    const excludeV2 = isMobileRequest && (semverAppVersion === null || semver.lt(semverAppVersion, fixVersion))
+    const excludeV2 = false
```

Comment:
if this is always to false shall we delete it beneath too?

Reply:
We can't delete it - we need the check for it. We would need to completely remove the condition.

I left it on purpose to reflect the mainnet condition.